logger/logger_test.go

```diff
@@ -117,17 +117,17 @@ func validateError(log string, has bool, t *testing.T) {
 	}
 }
 
-// validateError validates for warning logs.
-func validateWarning(log string, has bool, t *testing.T) {
-	// 1. Validate for Warning function.
-	Warning("warning log")
-	if has != search(log, t, wPrefix, "warning log") {
-		t.Errorf("warning entry mishmash [result: %v]", !has)
-	}
-	// 2. Validate for Warningf function.
-	Warningf("warning log [%v,%s]", "param1", "param2")
-	if has != search(log, t, wPrefix, "warning log [param1,param2]") {
-		t.Errorf("warningf entry mishmash: [result: %v]", !has)
+// validateError validates for warn logs.
+func validateWarn(log string, has bool, t *testing.T) {
+	// 1. Validate for Warn function.
+	Warn("warn log")
+	if has != search(log, t, wPrefix, "warn log") {
+		t.Errorf("warn entry mishmash [result: %v]", !has)
```

Comment:
`mishmash` - I guess that it's: `mismatch`, please do find/replace of this typo

Reply:
I will do it. It's a rather funny typo, and I agree we should fix it.